TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Components/Gyro.java

```diff
@@ -0,0 +1,68 @@
+package org.firstinspires.ftc.teamcode.Components;
+
+import com.qualcomm.hardware.bosch.BNO055IMU;
+import com.qualcomm.hardware.bosch.JustLoggingAccelerationIntegrator;
+import com.qualcomm.robotcore.eventloop.opmode.OpMode;
+
+import org.firstinspires.ftc.robotcore.external.navigation.AngleUnit;
+import org.firstinspires.ftc.robotcore.external.navigation.AxesOrder;
+import org.firstinspires.ftc.robotcore.external.navigation.AxesReference;
+import org.firstinspires.ftc.robotcore.external.navigation.Orientation;
+
+import java.util.Locale;
+
+public class Gyro {
+
+    OpMode opmode;
+    BNO055IMU imu;
+    Orientation angles;
+
+    public Gyro(OpMode opmode){
+        this.opmode = opmode;
+    }
+
+    public void init(boolean debug) {
+        BNO055IMU.Parameters parameters = new BNO055IMU.Parameters();
+        parameters.angleUnit = BNO055IMU.AngleUnit.DEGREES;
+        parameters.accelUnit = BNO055IMU.AccelUnit.METERS_PERSEC_PERSEC;
+        parameters.calibrationDataFile = "BNO055IMUCalibration.json"; // see the calibration sample opmode
+        parameters.loggingEnabled = true;
+        parameters.loggingTag = "IMU";
+        parameters.accelerationIntegrationAlgorithm = new JustLoggingAccelerationIntegrator();
+
+        // Retrieve and initialize the IMU. We expect the IMU to be attached to an I2C port
+        // on a Core Device Interface Module, configured to be a sensor of type "AdaFruit IMU",
+        // and named "imu".
+        imu = opmode.hardwareMap.get(BNO055IMU.class, "imu");
+        imu.initialize(parameters);
+
+        if(debug){
+            opmode.telemetry.addData("INFO","Gyro INIT");
+        }
+    }
+
```

Comment:
Last year we did this with quaternion math
https://github.com/MontclairRobotics/RelicRecovery2017/blob/141fed4c84d9de77ee8b0433a2fed3d638e41f9c/TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Gyro.java#L37
I think Josh did it, but we could just reuse his code.

Also, if we keep doing it this way, split this method up into different ones. Preferably a getX/Yaw, getY/Roll, getZ/Pitch (not sure if that's in the right order, but you know what I mean). and a debug() method